core/src/main/java/org/apache/brooklyn/core/objs/AbstractConfigurationSupportInternal.java

```diff
@@ -146,7 +146,6 @@ public T call() {
                 .immediately(true)
                 .deep(true)
                 .context(getContext())
-                .swallowExceptions()
```

Comment:
yeah, the immediate stuff is quite new so i'd prefer to try this.  if it's a problem we should perhaps look at wrapping in a `ReferenceWithError` as above, then caller will be forced to deal with the failure in the appropriate way.